include/aa_ddtr_strategy.hpp

```diff
@@ -94,6 +94,16 @@ namespace astroaccelerate {
     const int* ndms_data() const {
       return m_ndms.data();
     }
+
+    /** \return number of nsamples in the first range */
+    int t_samples() const {
```

Comment:
Rename function to nSamples_in_first_chunk.

Reply:
The function is not used anymore. Delete the t_samples.